<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widgetPlot_pie_1" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>171</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>false</underline>
    </font>
   </property>
  </widget>
  <widget class="QWidget" name="widgetPlot_pie_2" native="true">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>171</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>false</underline>
    </font>
   </property>
  </widget>
  <widget class="QWidget" name="widgetPlot_pie_3" native="true">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>171</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>false</underline>
    </font>
   </property>
  </widget>
  <widget class="QWidget" name="widgetPlot_pie_4" native="true">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>40</y>
     <width>171</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>false</underline>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
